sdk/identity/azure-identity/tests/test_shared_cache_credential_async.py

```diff
@@ -463,9 +452,7 @@
         await credential.get_token("scope")
     # error message should indicate multiple matching accounts, and list discovered accounts
```

Comment:
This comment should be updated or removed because we no longer expect the message to list discovered accounts.